### DISCUSSION TITLE
Netbox docker update

### DISCUSSION THREAD

**Discussion by rtrexmass** [netbox-community/netbox-docker#855] [Getting Help] [answered]
Hello, 

I had netbox installed with this docker repository, as v3.3.2, then i noticed the banner on the homepage notifying the new version v3.3.5.
![Screenshot_4](https://user-images.githubusercontent.com/105231805/194143016-09203016-f351-4f6a-8809-20e05699990c.png)

Then I did:

git stash
git pull origin release
docker-compose rm -fs netbox netbox-worker
docker-compose pull
docker-compose up -d netbox netbox-worker

That seems to upgrade me to v3.3.4. i just wanna confirm that what i did was correct, does the current version of docker is running behind the master project?

**Comment by tobiasge** [comment, marked as answer]
The Docker images are build every 24 hours. So it will take a while for a new version to be available.

**Reply by rtrexmass** [reply]
Oh i see, didnt notice that v3.3.5 came out today, Thanks :D
